build_tools/scripts/run_markdownlint.sh

```diff
@@ -15,13 +15,21 @@ declare -a included_files_patterns=(
   # All .md files (disabled while we decide how rigorously to apply lint checks)
   # "./**/*.md"
 
-  # Just .md files for the user-facing website.
```

Comment:
On purpose?

Reply:
Yeah? I'm expanding what files are covered by the lint check